Source/Core/Common/SDCardUtil.cpp

```diff
@@ -187,12 +187,12 @@ static unsigned int write_empty(FILE* file, u64 count)
 		if (len > count)
 			len = count;
 
-		if ( fwrite(empty, 1, (size_t)len, file) != (size_t)len )
-			return 1;
+		if (!file->WriteBytes(empty, (size_t)len))
```

Comment:
How useful is casting `len` to `size_t`, when it should technically be a `size_t` already (except that we store it as `u64` above)? Also, would it make sense to use `ArraySize` or even `std::array` there; while we're at it?

Altho, both those points are probably cleanup and don't belong in this PR...